vllm_ascend/worker/model_runner_v1.py

```diff
@@ -1605,9 +1605,12 @@ def execute_model(
                                                      intermediate_tensors))
         kv_connector_output = None
         if not vllm_version_is("0.10.0"):
-            kv_connector_output = KVConnectorOutput(
-                finished_sending=finished_sending,
-                finished_recving=finished_recving)
+            if finished_sending is not None and finished_recving is not None:
```

Comment:
https://github.com/vllm-project/vllm/commit/796bae07c59716b7b61d57343826bfbeabdd01bb added the check for kv_connector_output, we should ensure kv_connector_output  is None if not in PD case.